media-libs/mesa/mesa-17.0.3.ebuild

```diff
@@ -42,21 +42,23 @@ done
 IUSE="${IUSE_VIDEO_CARDS}
 	bindist +classic d3d9 debug +dri3 +egl +gallium +gbm gles1 gles2 +llvm
 	+nptl opencl osmesa pax_kernel openmax pic selinux vaapi valgrind vdpau
-	vulkan wayland xvmc xa"
+	vulkan wayland +X xvmc xa"
 
 REQUIRED_USE="
 	d3d9?   ( dri3 gallium )
+	dri3?   ( X )
 	llvm?   ( gallium )
 	opencl? ( gallium llvm )
-	openmax? ( gallium )
+	openmax? ( X gallium )
 	gles1?  ( egl )
 	gles2?  ( egl )
-	vaapi? ( gallium )
-	vdpau? ( gallium )
+	vaapi? ( X gallium )
+	vdpau? ( X gallium )
```

Comment:
I'm not sure why openmax/vaapi/vdpau would require X. Do they not compile without some X dependency?